observability/tracing/README.md

```diff
@@ -0,0 +1,103 @@
+# Distributed tracing
+
+With program flow distributed across many microservices, the ability to observe the different services in context can become a valuable tool for development and operations. 
+
+The simplest way to get an overview is to observe the Istio service mesh using Kiali. If there is a need to observe the causal relationship among individual calls to services, [envoy-based tracing](#envoy-based-context-extraction) is required.
+
+## Visualizing service mesh with Kiali
+
+Kiali can be reached at kiali.*cluster-name*.nais.io, eg. [kiali.dev-gcp.nais.io](https://kiali.dev-gcp.nais.io).
```

Comment:
Mention that kiali is _only_ for GCP clusters maybe?

Reply:
Yup!